lib/queue_classic.rb

```diff
@@ -100,7 +100,10 @@ def self.measure(data)
   # This will unlock all jobs any postgres' PID that is not existing anymore
   # to prevent any infinitely locked jobs
   def self.unlock_jobs_of_dead_workers
-    default_conn_adapter.execute("UPDATE #{QC.table_name} SET locked_at = NULL, locked_by = NULL WHERE locked_by NOT IN (SELECT pid FROM pg_stat_activity);")
+    server_version_num = default_conn_adapter.execute("SHOW server_version_num;")["server_version_num"]
```

Comment:
can we push this down into the conn adapter? Something along the lines of `#server_version`. If I remember correctly there is such a method provided on the `pg` connection.